frontend/src/components/SideNav.tsx

```diff
@@ -458,11 +461,11 @@ export class SideNav extends React.Component<SideNavInternalProps, SideNavState>
             disableHoverListener={!collapsed}
             disableTouchListener={!collapsed}
           >
-            <Link id='archiveBtn' to={RoutePage.ARCHIVE} className={commonCss.unstyled}>
+            <Link id='archiveBtn' to={RoutePage.ARCHIVED_RUNS} className={commonCss.unstyled}>
               <Button
                 className={classes(
                   css.button,
-                  page === RoutePage.ARCHIVE && css.active,
+                  page === RoutePage.ARCHIVED_RUNS && css.active,
```

Comment:
same, when page is archived experiments, also show this as active

Reply:
done